lms/djangoapps/api_manager/users/tests.py

```diff
@@ -39,8 +41,15 @@
 
 from django.contrib.auth.models import User
 from openedx.core.djangoapps.user_api.models import UserPreference
+from openedx.core.djangoapps.user_api.models import UserPreference
```

Comment:
This line is duplicated, as are some of these other imports. Other changed imports are not being used.